python/Ganga/new_tests/Unit/TestGangaObject.py

```diff
@@ -296,9 +296,9 @@ def write_read(thread_number):
                     child_num = rand.randint(0, 1000)
                     o.b.a = child_num
                     time.sleep(rand.uniform(0, 1E-6))
-                    self.assertEqual(o.a, num)
```

Comment:
pytest has a feature called [assertion introspection](https://pytest.org/latest/assert.html) whereby it can understand plain assert statements and provide very detailed feedback about the failure. Within `unittest.TestCase` subclasses we can continue to use `self.assertEqual()` but this allows us to write tests unconstrained by `unittest`.